subsys/net/lib/coap/coap_transport_socket.c

```diff
@@ -176,6 +176,32 @@ static session_t *session_find(const struct sockaddr *local,
 }
 #endif /* (COAP_SESSION_COUNT > 0) */
 
+
+/**@brief Internal method to find the session or port corresponding to the handle.
+ *
+ * @param handle Identifies the socket descriptor, also the the handle for the transport.
+ *
+ * @retval Pointer to a valid transport in case of success, else, NULL.
+ */
+static int32_t local_endpoint_find(coap_transport_handle_t  handle)
```

Comment:
Double space

Reply:
And we need to use Zephyr types here or CI will complain (I guess plain `int` should do the job there as well).